From: Emmanuel Bourg <ebourg@apache.org>
Date: Sat, 25 Dec 2021 23:34:46 +0100
Subject: Modified the assembly plugin configuration to use
 debian/assembly-config.xml

Forwarded: not-needed

instead of the 'config' descriptor from the (missing) jetty-assembly-descriptors artifact
---
 jetty-cdi/pom.xml                                   | 6 +++---
 jetty-websocket/javax-websocket-server-impl/pom.xml | 6 +++---
 pom.xml                                             | 6 +++---
 3 files changed, 9 insertions(+), 9 deletions(-)

--- a/jetty-cdi/pom.xml
+++ b/jetty-cdi/pom.xml
@@ -70,9 +70,9 @@
               <goal>single</goal>
             </goals>
             <configuration>
-              <descriptorRefs>
-                <descriptorRef>config</descriptorRef>
-              </descriptorRefs>
+              <descriptors>
+                <descriptor>../debian/assembly-config.xml</descriptor>
+              </descriptors>
             </configuration>
           </execution>
         </executions>
--- a/jetty-websocket/javax-websocket-server-impl/pom.xml
+++ b/jetty-websocket/javax-websocket-server-impl/pom.xml
@@ -80,9 +80,9 @@
               <goal>single</goal>
             </goals>
             <configuration>
-              <descriptorRefs>
-                <descriptorRef>config</descriptorRef>
-              </descriptorRefs>
+              <descriptors>
+                <descriptor>../../debian/assembly-config.xml</descriptor>
+              </descriptors>
             </configuration>
           </execution>
         </executions>
--- a/pom.xml
+++ b/pom.xml
@@ -1370,9 +1370,9 @@
                   <goal>single</goal>
                 </goals>
                 <configuration>
-                  <descriptorRefs>
-                    <descriptorRef>config</descriptorRef>
-                  </descriptorRefs>
+                  <descriptors>
+                    <descriptor>debian/assembly-config.xml</descriptor>
+                  </descriptors>
                 </configuration>
               </execution>
             </executions>
